flow/controllers/velocity_controllers.py

```diff
@@ -25,11 +25,10 @@ class FollowerStopper(BaseController):
     def __init__(self,
                  veh_id,
                  car_following_params,
-                 v_des=15,
-                 danger_edges=None):
+                 v_des=15):
         """Instantiate FollowerStopper."""
         BaseController.__init__(
-            self, veh_id, car_following_params, delay=1.0,
+            self, veh_id, car_following_params, delay=0.0,
```

Comment:
Not sure why the followerstopper controller in Flow was implemented with a delay. Definitely is not necessary